src/types/api.js

```diff
@@ -17,6 +36,31 @@
  * @prop {string} text
  * @prop {string} uri
  * @prop {string} user
+ * @prop {boolean} hidden
+ *
+ * @prop {Object} document
+ *   @prop {string} document.title
```

Comment:
I'm not sure if the `document` and `document.title` properties will always be present. You'd have to check the backend code to find out.

Reply:
I'll check on this and change if need.